src/client/api/tests/SConscript

```diff
@@ -8,6 +8,9 @@ def scons():
     daos_build.test(denv, 'eq_tests', Glob('eq_tests.c'),
                     LIBS=['daos', 'daos_common', 'gurt', 'cart',
                           'pthread', 'cmocka'])
+    daos_build.test(denv, 'agent_tests', Glob('agent_tests.c'),
+                    LIBS=['daos', 'daos_common', 'gurt', 'cart',
```

Comment:
Are pthread and cart needed by libdaos? Doesn't seem like these tests would need them

Reply:
I believe they are. Unfortunately we need to include them I believe.